### PR TITLE
adding translation feature

### PR DESCRIPTION
**Pull request by mrai-mm**
https://mademedia.freshdesk.com/a/tickets/85664

### CODE REVIEW

File: data/translations.json [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    "cy": "G"
+  },
+  "day_saturday": {
+    "en": "S",
+    "cy": "S"
+  },
+  "day_sunday": {
+    "en": "S",
+    "cy": "S"
+  }
```
Comment (lbradley-mm) [inline data/translations.json:77]: These don't seem to be used

**Review by lbradley-mm — COMMENTED** [review]
The `day_xxx` values aren't being used

File: calus.js [hunk 2]
```diff
@@ -22,7 +23,7 @@ let defaultTemplate = `
                 ‹
             </button>
             <div class="month__text">
-                {{ month.time.toFormat(month.isInCurrentYear ? 'MMMM' : 'MMMM y') }}
+                {{ translate('month_' + month.time.toFormat('MMMM'), 'cy') }} {{ month.time.toFormat(month.isInCurrentYear ? '' : 'y') }}
```
Comment (lbradley-mm) [inline calus.js:26]: You're passing `cy` in but what about when it's something different

**Review by lbradley-mm — COMMENTED** [review]
See comment

File: data/translations.json (hunk 1 above)
Comment (mrai-mm) [reply data/translations.json:77]: These will be used in the WMC template.

File: calus.js (hunk 2 above)
Comment (mrai-mm) [reply calus.js:26]: Actually, I will force it to default to `English` if there are no language. Language will always be passed from the SS template.

**Review by lbradley-mm — APPROVED** [review]

**Pull request merged**